cluster-autoscaler/cloudprovider/gce/autoscaling_gce_client.go

```diff
@@ -195,6 +198,24 @@ func (client *autoscalingGceClientV1) ResizeMig(migRef GceRef, size int64) error
 	return client.waitForOp(op, migRef.Project, migRef.Zone, false)
 }
 
+func (client *autoscalingGceClientV1) CreateInstances(migRef GceRef, delta int64, existingInstances []string) error {
+	registerRequest("instance_group_managers", "create_instances")
+	req := gce.InstanceGroupManagersCreateInstancesRequest{}
+	instanceNames := map[string]bool{}
+	for _, inst := range existingInstances {
+		instanceNames[inst] = true
+	}
+	req.Instances = make([]*gce.PerInstanceConfig, 0, delta)
+	for i := int64(0); i < delta; i++ {
+		req.Instances = append(req.Instances, &gce.PerInstanceConfig{Name: generateInstanceName(migRef, instanceNames)})
```

Comment:
I know it probably doesn't matter that much, but for completeness sake could you also add the newly created instance names to the map?

Reply:
good point, added update to the map